client/driver/executor/executor_windows.go

```diff
@@ -0,0 +1,111 @@
+// +build windows
+
+package executor
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"syscall"
+
+	syslog "github.com/RackSec/srslog"
+
+	"github.com/hashicorp/nomad/client/driver/logging"
+)
+
+func (e *UniversalExecutor) LaunchSyslogServer() (*SyslogServerState, error) {
```

Comment:
Like you pointed out in the issue description I don't think there was any reason for this to be in the `executor_unix.go` file instead of `executor.go` proper. Would you mind moving it there to avoid duplication?